webrtc-android-sample-app/src/main/java/io/antmedia/webrtc_android_sample_app/ScreenCaptureActivity.java

```diff
@@ -100,11 +101,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 return;
             }
         }
+        // fix:- to remove the black boarder issue
+        DisplayMetrics displayMetrics = webRTCClient.getDisplayMetrics();
+        videoWidth = displayMetrics.widthPixels;
+        videoHeight = displayMetrics.heightPixels;
 
         this.getIntent().putExtra(CallActivity.EXTRA_CAPTURETOTEXTURE_ENABLED, true);
         this.getIntent().putExtra(CallActivity.EXTRA_VIDEO_BITRATE, 2500);
-        this.getIntent().putExtra(CallActivity.EXTRA_VIDEO_WIDTH, 540);
-        this.getIntent().putExtra(CallActivity.EXTRA_VIDEO_HEIGHT, 960);
+        this.getIntent().putExtra(CallActivity.EXTRA_VIDEO_WIDTH, videoWidth);
```

Comment:
High resolution may degrade the video quality and increase the pixelation so we may need to decrease the resolution by keeping the aspect ratio

FYI